tests/common/plugins/memory_utilization/README.md

```diff
@@ -0,0 +1,573 @@
+### Scope
```

Comment:
@lipxu  could we have a section as use case, and give introduction how to use this fixture?

1. To define a memory check globally. 
2. To define a memory check per HWSKU
3. To define a memory check per test module/test case
4. When different memory check defined for global, HWSKU, test cases, how them interact?

Reply:
Sure, updated the README , add "Example Use Cases for memory items" section, thanks for reminder, please review again.